lib/importer/factory/collection_factory.rb

```diff
@@ -0,0 +1,18 @@
+module Importer
+  module Factory
+    class CollectionFactory < ObjectFactory
+      self.klass = Collection
+      self.system_identifier_field = :identifier
```

Comment:
In the other factories, this line has a helpful comment. Missing in the `CollectionFactory`.